<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acceuil</class>
 <widget class="QDialog" name="acceuil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>681</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>681</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Medecin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#001328;

</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>400</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{background:#738F84;
color:#0B3B57;
border:1px solid #738F84;
border-radius:12px;
}
*:hover{
background:#B5DEFA;
opacity:1;
border:1px solid;
color:#08828F;
transition:2s infinite;
border-radius:12px;
}


</string>
     </property>
     <property name="text">
      <string>LISTE PATIENT TRAITER PAR UN MEDECIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{background:#738F84;
color:#0B3B57;
border:1px solid #738F84;
border-radius:12px;
}
*:hover{
background:#B5DEFA;
opacity:1;
border:1px solid;
color:#08828F;
transition:2s infinite;
border-radius:12px;
}


</string>
     </property>
     <property name="text">
      <string>LISTE DES MEDECINS CONSULTER PAR UN PATIENT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>490</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*
{background:#738F84;
color:#0B3B57;
border:1px solid #738F84;
border-radius:12px;
}
*:hover{
background:#B5DEFA;
opacity:1;
border:1px solid;
color:#08828F;
transition:2s infinite;
border-radius:12px;
}


</string>
     </property>
     <property name="text">
      <string>ETAT DES PRESTATIONS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>601</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>     Sous l' Organiisation de l' OMS(Organisation Mondiale de la Santé)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:#08828F;

</string>
     </property>
     <property name="text">
      <string>  GESTION DES PRESTATIONS MEDECINS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">images:url(:/img/icons8_shutdown.ico);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/img/icons8_shutdown.ico</normaloff>:/img/icons8_shutdown.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>33</width>
       <height>56</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelimage">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>531</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonoms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">images:url(:/img/icons8_shutdown.ico);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/img/schemaOrgLogo.jpg</normaloff>:/img/schemaOrgLogo.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>58</width>
       <height>65</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>medecin</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="title">
      <string>Liste des medecins</string>
     </property>
     <widget class="QTableView" name="tableView_med">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>290</y>
       <width>291</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_23">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;
color:white;
font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_24">
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(255, 255, 94);

font: bold i large &quot;TimesbNew Roman&quot;;

</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_25">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;
color:white;
font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_26">
        <property name="styleSheet">
         <string notr="true">color:black;

font: bold i large &quot;TimesbNew Roman&quot;;

background-color:rgb(255, 255, 94);</string>
        </property>
        <property name="text">
         <string>Recherche des Medecins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_27">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;
color:white;
font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Mise à jour Liste</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelpicmed">
     <property name="geometry">
      <rect>
       <x>376</x>
       <y>10</y>
       <width>441</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>patient</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>290</y>
       <width>291</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_36">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;
color:white;
font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_37">
        <property name="styleSheet">
         <string notr="true">color:black;
background-color:rgb(255, 255, 94);

font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_38">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;
color:white;

font: bold i large &quot;TimesbNew Roman&quot;;
</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_39">
        <property name="styleSheet">
         <string notr="true">color:black;

font: bold i large &quot;TimesbNew Roman&quot;;

background-color:rgb(255, 255, 94);</string>
        </property>
        <property name="text">
         <string>Recherche des Patients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_40">
        <property name="styleSheet">
         <string notr="true">background-color:#5E64A6;

font: bold i large &quot;TimesbNew Roman&quot;;

color:white;</string>
        </property>
        <property name="text">
         <string>Mise à jour Liste</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="title">
      <string>Liste des Patients</string>
     </property>
     <widget class="QTableView" name="tableView_pat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>321</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelpicpat">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>441</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>traitement</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>791</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
</string>
     </property>
     <property name="title">
      <string>Liste des Traitements</string>
     </property>
     <widget class="QTableView" name="tableView_traite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>771</width>
        <height>311</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_41">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#5E64A6;
color:white;
font: bold i large &quot;TimesbNew Roman&quot;;
</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_42">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background-color:rgb(255, 255, 94);

font: bold i large &quot;TimesbNew Roman&quot;;
</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_43">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>450</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#5E64A6;
color:white;

font: bold i large &quot;TimesbNew Roman&quot;;
</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_45">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background-color:rgb(255, 255, 94);

font: bold i large &quot;TimesbNew Roman&quot;;
</string>
     </property>
     <property name="text">
      <string>Mise à jour Liste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_44">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>500</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;

font: bold i large &quot;TimesbNew Roman&quot;;

background-color:rgb(255, 255, 94);</string>
     </property>
     <property name="text">
      <string>Recherche des Traitements</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
